modules/nextflow/src/main/groovy/nextflow/container/SarusBuilder.groovy

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020-2022, Seqera Labs
+ * Copyright 2013-2019, Centre for Genomic Regulation (CRG)
```

Comment:
I think Pawsay should be reported instead of CRG 😉

Reply:
done!